README.md

```diff
@@ -13,7 +13,7 @@ Besu is an Apache 2.0 licensed, MainNet compatible, Ethereum client written in J
 
 ## Issues 
 
-Besu issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10117) not GitHub. 
+Besu issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10117), not GitHub. 
```

Comment:
@fakhan11 the README has been refactored (moved) and this change was included over there. Could you close this PR? Thanks